app/app.go

```diff
@@ -59,7 +59,7 @@ var (
 		supply.AppModuleBasic{},
 
 		// Engine's AppModuleBasic
-		cosmos.NewAppModuleBasic(ownershipsdk.ModuleName),
+		cosmos.NewAppModuleBasic(ownership.ModuleName),
```

Comment:
We don't want to use any custom code like we previously did. Don't call `cosmos.NewAppModuleBasic` as it uses the moduleBasic define in `cosmos/module.go` and don't implement much stuff, as opposed as your new version in `x/ownership/module.go`
```suggestion
		ownership.AppModuleBasic{},
```